### DISCUSSION TITLE
Test with pandera

### DISCUSSION THREAD

**Discussion by hurtado-santiago** [pydantic/pydantic#2850] [Feature Request]
Hi!

Can you include a test with [pandera](https://pandera.readthedocs.io/en/stable/index.html#)? 
I would like to see which is faster. 

Thanks!

**Comment by samuelcolvin** [comment]
I assume you mean include in the benchmarks?

Unless I'm missing something, pandera is doing something slightly different, not sure how it would fix into the benchmarks. 

Either way, I'm afraid I try to avoid adding other packages unless they're popular and we'll known, I'm not sure pandera has a sufficiently big following.

**Reply by hurtado-santiago** [reply]
Yes, I was referring to the benchmark. 

could you tell me a little the difference between pandera and pydantic?

Thanks for your answer
